<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Color_Calibration</class>
 <widget class="QMainWindow" name="Color_Calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="colors">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1061</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>427</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_button2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QListWidget" name="color_list">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>131</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="color_insert">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="frame_clusterizado">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>427</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clusterize_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CLUSTERIZE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
